### PR TITLE
Cgreen based unit tests for gvmd

### PR DESCRIPTION
**Pull request by mattmundell**
Exploring Cgreen as an option for gvmd unit tests.

### CODE REVIEW

**Comment by bjoernricks** [comment]
:+1: :+1: :+1: really nice!

File: src/gmp_tickets_tests.c [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  GError *error;
+  const gchar *uuid;
+
+  /* Check that create_ticket_run calls only copy_ticket when given COPY. */
+
+  uuid = "9b5da19e-86b4-11e9-b0d2-28d24461215b";
+  gmp_parser.client_writer = dummy_client_writer;
+
+  /* <CREATE_TICKET> */
+  create_ticket_start (&gmp_parser, NULL, NULL);
```
Comment (timopollmeier) [inline src/gmp_tickets_tests.c]: Passing NULL pointers for attribute_names and attribute_values in the create_ticket_start(...) call seems to be causing the segfault for me.

Comment (mattmundell) [reply src/gmp_tickets_tests.c]: Oh, I forgot that I patched gvmd-libs: greenbone/gvm-libs/pull/240

Comment (timopollmeier) [reply src/gmp_tickets_tests.c]: With the gvm-libs patch it works for me.

**Pull request merged**